src/site/content/en/mini-apps/mini-app-example-project/index.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+title: An example project
+authors:
+  - thomassteiner
+date:  2021-02-05
+# updated: 2021-01-25
+description: |
+  This chapter shows an example project that follows the "programming the mini app way" approach.
+tags:
+  - mini-apps
+---
+
+## The app domain
+
```

Comment:
Needs an intro

Reply:
I think you should link here to the other pages that explain the high-level concepts of mini app programming

Reply:
Done.